helpers.go

```diff
@@ -59,6 +59,26 @@ func AssertEqualAPIError(t require.TestingT, err error, expected ServerResponse)
 	return assert.Equal(t, expected.Error, errorField.String())
 }
 
+func AssertAPIError(t require.TestingT, actual error, expectedCode int, expectedFormat string, a ...interface{}) bool {
+	if n, ok := t.(interface {
```

Comment:
We already have these checks in `AssertEqualAPIError` Why don't we just update that method?

Reply:
Only because that would force us to change all tests in a single commit. If you are ok with that – go ahead. :)